### ISSUE TITLE
Support versioned tags 

### ISSUE THREAD

**Issue by ZerNox** [odoo/docker#385]
When you want to base this image as a multistage build it would be very useful to have a version on the odoo docker image. 

Suggestion: 
odoo:14.0-20210903

**Comment by arabkhemar** [comment]
@ZerNox you can add '"local" tag after pulling an image (version from Dockerfile)
`docker tag odoo:14.0 odoo:14.0-XXXXYYZZ`

**Comment by avkhozov** [comment]
+1 for this issue.

@arabkhemar You can't create identical environmental with local tags because odoo:14.0 can be different at different times. And you should use digests instead of tags, which are not convenient.

**Comment by avkhozov** [comment]
There is also a problem with getting a consistent instance with Odoo Enterprise in a docker environment https://github.com/odoo/docker/issues/392#issuecomment-1007707903. It seems that proper versioning can solve this problem as well.

**Comment by avkhozov** [comment]
@d-fence, is there a chance that docker's tags with odoo's releases will appear in the official registry https://hub.docker.com/_/odoo?

**Comment by amh-mw** [comment]
TLDR: [docker-library/repo-info](https://github.com/docker-library/repo-info) contains sufficient digests/versions for a decent workaround

If you check out the [history for repos/odoo](https://github.com/docker-library/repo-info/commits/master/repos/odoo), you will notice the commit messages run either "scan-local.sh" or "update-remote.sh".

The update remote changes contain docker digests:

> $ docker pull odoo@sha256:ea7a1bfeb448ea7d3a2a9724b2153bb409bfa5b0085bf18531d5e928eac8efd0

and the scan local changes contain package version updates:

> - `odoo=13.0.20220325`

This means that for any given commit, you can reasonably work around the lack of minor version tagging in Docker Hub with a short script:

```
export ODOO_DIGEST="sha256:ea7a1bfeb448ea7d3a2a9724b2153bb409bfa5b0085bf18531d5e928eac8efd0"
export ODOO_VERSION="13.0.20220325"
docker pull odoo@$ODOO_DIGEST
docker tag odoo@$ODOO_DIGEST odoo@$ODOO_VERSION
...rest of your build...
```

The information currently exists for Odoo versions 13.0 through 15.0 and the digests can be successfully pulled from Docker Hub.

**Comment by amh-mw** [comment]
Given that each image has a `major.minor.date` version Odoo package, it seems like it should be trivial to also tag the docker image with that version.

**Comment by ar-mw** [comment]
I see devs have been asking about this for over year along with offering possible fixes. Is anyone from Odoo readying these tickets?

**Comment by MartinVerges** [comment]
That is important,... please odoo, do this!
